src/librustc_mir/borrow_check/region_infer/mod.rs

```diff
@@ -202,7 +202,7 @@ pub(crate) enum Cause {
 ///
 /// For more information about this translation, see
 /// `InferCtxt::process_registered_region_obligations` and
-/// `InferCtxt::type_must_outlive` in `rustc::infer::outlives`.
+/// `InferCtxt::type_must_outlive` in `rustc_middle::infer::outlives`.
```

Comment:
Is that still there, wasn't `InferCtxt` moved?

Reply:
Was moved to `rustc_infer::infer::InferCtxt` says rustdoc nightly docs.